### ISSUE TITLE
Disabling the viewport when it would be disabled anyway causes a crash

### ISSUE THREAD

**Issue by jaydenmilne** [BYU-PCCL/holodeck#378]
**Describe the bug**
Passing in `show_viewport = False` to `make()` or `HolodeckEnvironment` causes a crash when it would already be running headless.

**Expected behavior**
It shouldn't crash

**Version Information:**
 - Holodeck Version < 0.3.0
 - World/Scenario version

change to `environment.pop('DISPLAY', None)`

```
Traceback (most recent call last):
  File "main.py", line 114, in <module>
    app.run(train_holodeck_ppo)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/venv/lib/python3.6/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/venv/lib/python3.6/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "main.py", line 52, in train_holodeck_ppo
    image_dir=FLAGS.image_dir)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/holodeck_env.py", line 24, in __init__
    show_viewport=False)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/venv/src/holodeck/src/holodeck/holodeck.py", line 91, in make
    return HolodeckEnvironment(**param_dict)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/venv/src/holodeck/src/holodeck/environments.py", line 101, in __init__
    show_viewport=show_viewport)
  File "/mnt/pccfs/backed_up/kolby/holodeck-baselines/venv/src/holodeck/src/holodeck/environments.py", line 647, in __linux_start_process__
    del environment['DISPLAY']
KeyError: 'DISPLAY'
```

**Comment by daniekpo** [comment]
Fixed in #392

**Comment by jaydenmilne** [comment]
✅ in changelog
